### PR TITLE
found a new ad host

### PR DESCRIPTION
**Pull request by Ar0xA**

### CODE REVIEW

**Review by arthurgeron — CHANGES_REQUESTED** [review]
yt3.gghpt is used to store images, has nothing to do with ads.
r5.sn-32o-guhl.googlevideo.com doesn't seem to break YT, still, I'm going to further test it.

File: hosts.txt [hunk 1]
```diff
@@ -76,3 +76,5 @@
 0.0.0.0 akamaitechnologies.com
 0.0.0.0 akamaiedge.net
 0.0.0.0 ocsp.comodoca.com
+0.0.0.0 yt3.ggpht.com
```
Comment (arthurgeron) [inline hosts.txt:79]: Should be removed

**Comment by viumvi** [comment]
also, noticed that blocking s.ytimg.com prevents normal functioning.

**Comment by arthurgeron** [comment]
I've added your suggestion (except for yt3.ggpht.com)
s.ytimg.com has been removed from the list

**Pull request closed without merging**